upup/pkg/fi/executor.go

```diff
@@ -134,7 +134,7 @@ func (e *executor) RunTasks(taskMap map[string]Task) error {
 
 				remaining := time.Second * time.Duration(int(time.Until(ts.deadline).Seconds()))
 				if _, ok := err.(*TryAgainLaterError); ok {
-					klog.Infof("Task %q not ready: %v", ts.key, err)
+					klog.V(2).Infof("Task %q not ready: %v", ts.key, err)
```

Comment:
This is a special case where things fail because something else is not ready due to how AWS works.
Usually you expect the tasks to succeed in the order they are and any retries imply some issue.

Reply:
I think if we're dealing with eventual consistency (what TryAgainLaterError seems to imply) then it is within the expected behavior of the system, and therefor isn't worth alerting to the user at the default verbosity levels that it has occurred. If tasks consistently fail or fail for different reasons those situations will still be logged normally.

I just don't think this is a valuable enough piece of information to surface to the user.

Reply:
I'd say you're probably both right ...

As a user, I don't much care about one or two failures here.  I probably do care if it gets "stuck" to know why it's stuck.  And I probably also want to know roughly what is going on ("waiting for 2 tasks" ?).

Maybe we should do something different on iteration 1 vs iteration 10?

Reply:
if a task is stuck then kops will keep retrying it until 10 minutes have elapsed and log a more descriptive error:

https://github.com/kubernetes/kops/pull/11080/files#diff-d73723c1c245d32c779df66e821830949e518a324bd48cfcf8299ef00a2194dfR105

kops also logs if no progress has been made on any remaining tasks. I think both of these are sufficient enough to warrant not needing to log the specific task that isn't ready. Users can always `-v 2` if they need more detail.